documentation/assemblies/assembly-address-space-examples.adoc

```diff
@@ -9,7 +9,7 @@ include::../modules/ref-address-space-example.adoc[leveloffset=+1]
 
 include::../modules/ref-address-space-example-auth-service.adoc[leveloffset=+1]
 
-include::../modules/ref-address-space-example-exposing-endpoints.adoc[leveloffset=+1]
```

Comment:
I dont think exposing example should be removed?

Reply:
@lulf Thanks for catching this.